app/views/applicant_details/_form.html.erb

```diff
@@ -69,15 +72,19 @@
   </div>
 
   <div class="field">
-    <%= form.label :birthdate%>
+    <%= form.label :birthdate %>
     <div class="date_select">
-      <%= form.date_select :birthdate, default: applicant_detail.birthdate, start_year: Date.today.year - 19, end_year: Date.today.year - 12, prompt: "Select",required: true %>
+      <%= form.text_field :birthdate, required: true, data: { 
+        controller: "flatpickr", 
+        flatpickr_min_date_value: (Date.today - 19.years).strftime('%Y-%m-%d'), 
+        flatpickr_max_date_value: (Date.today - 12.years).strftime('%Y-%m-%d') 
+      }, placeholder: "Select your birthdate" %>
     </div>
```

Comment:
In the edit form the value of the birthdate field is not displayed.

Reply:
I am seeing it.
Try doing a bin/rails db:reset to be sure you don't have bad data